frontend/src/slices/auth/actions.ts

```diff
@@ -14,14 +18,15 @@ import {
 } from '~/packages/users/users.js';
 
 import { name as sliceName } from './auth.slice.js';
+import { type CustomAsyncThunkConfig } from './libs/types/types.js';
 
 const signUp = createAsyncThunk<
   CustomerSignUpResponseDto | BusinessSignUpResponseDto,
   {
     payload: CustomerSignUpRequestDto | BusinessSignUpRequestDto;
     mode: ValueOf<typeof AuthMode>;
   },
-  AsyncThunkConfig
+  CustomAsyncThunkConfig
```

Comment:
Maybe give a clearer name for the type? Right now it is not clear from the name how it differs from `AsyncThunkConfig`.